app/controllers/package.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2023 HM Revenue & Customs
+ * Copyright 2024 HM Revenue & Customs
```

Comment:
I'm not sure if we want to change the date of this header - we might do though. Worth checking with Lukasz as he mentioned it before (can't remember what he said).

Reply:
I think the latest HMRC advice is to keep whatever date we have if we just update the file. The new date obviously for new files

Reply:
Thanks, good to know. Do you know why this would automatically change for any reason? I definitely didn't change the header intentionally.

Reply:
yep, 3 years ago when I was in HMRC that was the policy - some automatization updating copyright each time you change it. Now we just do it in new file. We'll learn how everything works after the SOW mongodb ;)

Reply:
Before it was changing because the `uk.gov.hmrc.sbt-auto-build` sbt plugin was out of date. Not sure why it would continue to do it now I have updated it.